plugins/remotefortressreader/remotefortressreader.cpp

```diff
@@ -193,7 +192,7 @@ const char* growth_locations[] = {
 #include "df/art_image.h"
 #include "df/art_image_chunk.h"
 #include "df/art_image_ref.h"
-command_result loadArtImageChunk(color_ostream &out, vector <string> & parameters)
+command_result loadArtImageChunk(color_ostream &out, std::vector<std::string> & parameters)
```

Comment:
it's fairly common, and acceptable by our standards, to import just `vector` and `string` with 'using std::string; using std::vector;`

Reply:
> it's fairly common, and acceptable by our standards, to import just `vector` and `string` with 'using std::string; using std::vector;`

I did consider resolving it that way, there are already multiple instances of `std::vector` being written before I made any changes so I opted for that way instead, personally I only do `using std::xxx` if some types are getting unweildy to write out which wasn't really the case here (there are only 18 usages of `std::xxx` in the entire file).